docs/pages/contribution/advanced-topics/hirarchical-layers.js

```diff
@@ -0,0 +1,85 @@
+// mainnet parameters
+const SLOTS_PER_EPOCH = 32;
+const SECONDS_PER_SLOT = 12;
+
+// Simulation parameters
+const simPeriod = 60 * 60 * 24 * 30; // 30 days
+const S_full = 1024 * 1024 * 1.8;
+const S_diff = 1024 * 48;
+const T_full = 100;
+const T_diff = 150;
+const T_replay = 100;
+const R_full = 10;
+const R_diff = 50;
+const W_s = 1;
```

Comment:
This is a big assumption that we weight 1) total storage size 2) total backup time 3) total restore backup time on a 1:1:1 ratio when calculating the cost. Might be okay for now

Reply:
Yes it's assumption can only be concrete with more data that we collect in real usage.